hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.test.tsx

```diff
@@ -27,6 +27,40 @@ jest.mock("api/api", () => ({
 
 const mockedPost = post as jest.MockedFunction<typeof post>;
 
+const mockHardwareSignOutStartDate = jest.fn();
+const mockHardwareSignOutEndDate = jest.fn();
+jest.mock("constants.js", () => ({
+    get hardwareSignOutStartDate() {
+        return mockHardwareSignOutStartDate();
+    },
+    get hardwareSignOutEndDate() {
+        return mockHardwareSignOutEndDate();
+    },
+    get minTeamSize() {
```

Comment:
<img width="590" alt="CleanShot 2022-08-21 at 15 28 00@2x" src="https://user-images.githubusercontent.com/36619969/185807660-ff61eb1d-0e39-4eda-9403-07a17ebb3203.png">

According to Logistic team, the minimum team size should be 3

Reply:
Oh Leo this is just for tests, it's not the actual constant value so it can be anything. Also this ticket was meant to update dates only, I will address all the team information in my ticket (NH-13)